gordo_components/model/metadata.py

```diff
@@ -0,0 +1,137 @@
+from contextlib import contextmanager
+from typing import Any, Optional
+
+import threading
+import warnings
+
+
+class MetadataCollector(object):
```

Comment:
`(object)` is not necessary.

Reply:
:open_mouth:  ... there wasn't a mind blown emoji.